### ISSUE TITLE
Write a JAX optimizer notebook

### ISSUE THREAD

**Issue by yupinghuang** [yupinghuang/jax-ri#2]
See https://jax.readthedocs.io/en/latest/jax.experimental.optimizers.html

Research a few of these and  use them on a toy problem (linear regression?)

Maybe try this as well to get the Bayesian MAP estimator https://ui.adsabs.harvard.edu/abs/2020arXiv200514199H/abstract.

**Comment by yupinghuang** [comment]
the optimizer wrapper codes can also go in https://github.com/yupinghuang/jax-ri/blob/main/jaxri/optimize.py. I left some pointers there for using the optimizers that come with JAX. Should do a gradient descent too.

**Comment by yupinghuang** [comment]
Now there's stuff in `lnprob.py` and `priors.py` to get gradients. A test array configuration (ok this is not realistic) is at https://github.com/yupinghuang/jax-ri/blob/main/notebooks/random_baselines.csv for testing.
